A66-otel-stats.md

```diff
@@ -0,0 +1,540 @@
+  "go.opentelemetry.io/otel/metric"
+)
+
+
+package opentelemetry
+
+// MetricsOptions are the metrics options for OpenTelemetry instrumentation.
+type MetricsOptions struct {
+  // MeterProvider is the MeterProvider instance that will be used for access
+  // to Named Meter instances to instrument an application. To enable metrics
+  // collection, set a meter provider. If unset, no metrics will be recorded.
+  // Any implementation knobs (i.e. views, bounds) set in the passed in object
+  // take precedence over the API calls from the interface in this component
+  // (i.e. it will create default views for unset views).
+  MeterProvider metric.MeterProvider
+
+  // Metrics are the metrics to instrument. Will turn on the corresponding
+  // metric supported by the client and server instrumentation components if
+  // applicable.
+  Metrics []string
+
+  // Attributes are constant attributes applied to every recorded metric.
+  Attributes []attribute.KeyValue
+}
+
+// DialOption returns a dial option which enables OpenTelemetry instrumentation
+// code for a grpc.ClientConn.
+//
+// Client applications interested in instrumenting their grpc.ClientConn should
+// pass the dial option returned from this function as a dial option to
+// grpc.Dial().
+//
+// For the metrics supported by this instrumentation code, a user needs to
+// specify the client metrics to record in metrics options. A user also needs to
+// provide an implementation of a MeterProvider. If the passed in Meter Provider
+// does not have the view configured for an individual metric turned on, the API
+// call in this component will create a default view for that metric.
+func DialOption(mo MetricsOptions) grpc.DialOption {}
+
+// ServerOption returns a server option which enables OpenTelemetry
+// instrumentation code for a grpc.Server.
+//
+// Server applications interested in instrumenting their grpc.Server should pass
+// the server option returned from this function as an argument to
+// grpc.NewServer().
+//
+// For the metrics supported by this instrumentation code, a user needs to
+// specify the client metrics to record in metrics options. A user also needs to
+// provide an implementation of a MeterProvider. If the passed in Meter Provider
+// does not have the view configured for an individual metric turned on, the API
+// call in this component will create a default view for that metric.
+func ServerOption(mo MetricsOptions) grpc.ServerOption {}
+```
+
+#### Python
+
+To be filled
+
+### Migration from OpenCensus
+
+The following sections show the differences between the gRPC OpenCensus spec and
+the proposed gRPC OpenTelemetry spec and the mapping of metrics between the two.
+It also presents metrics present in OpenCensus spec that do not map to a metric
+in the OpenTelemetry spec at present. Two migration strategies are also proposed
+for customers who are satisfied with the stats coverage provided by the current
+OpenTelemetry spec.
```

Comment:
Do we mean `current OpenCensus spec` here?

Reply:
No, the `current OpenTelemetry spec` refers to this spec. Updated the wording.